src/dss.jl

```diff
@@ -0,0 +1,5774 @@
+module lib
+
+using ..OpenDSSDirect
```

Comment:
Is this needed?

Reply:
It's `LIBRARY`, right? I'd do `import ..OpenDSSDirect: LIBRARY`.

Reply:
Good idea! I didn't think of that! Fixed in 3259111 and f1bc5ec.